coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
Good point. Actually I'll just remove `LOG_LEVEL.xxx` from everywhere, and use directly `logging.xxx`. There's no point of extra redirection.